envoy/http/alternate_protocols_cache.h

```diff
@@ -128,9 +142,19 @@ class AlternateProtocolsCache {
    * @return the number if entries in the map.
    */
   virtual size_t size() const PURE;
+
+  /**
+   * @param origin The origin to get HTTP/3 status tracker for.
+   * @return the existing status tracker or creating a new one if there is none.
+   *         If the origin doesn't have a cached entry, creating the entry to hold the status
```

Comment:
I think I would remove this sentence as the previous sentence seems to make the behavior clear.